.github/workflows/build_and_test.yml

```diff
@@ -455,7 +455,7 @@ jobs:
         java-version: 11
     - name: Build with Maven
       run: |
-        export MAVEN_OPTS="-Xss64m -Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
+        export MAVEN_OPTS="-Xss2m -Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
```

Comment:
This is decreasing stack size, isn't it?

Reply:
Oh you're right. My intention was to up it to 2g here. What do you think is a good value to try?

I don't want to overshoot it, but probably `-Xss256m` will be sufficient?